shared/reducers/fs.js

```diff
@@ -142,10 +142,13 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
     }
     case FsGen.journalUpdate: {
       const {syncingPaths, totalSyncingBytes, endEstimate} = action.payload
-      return state
-        .setIn(['uploads', 'syncingPaths'], I.Set(syncingPaths))
-        .setIn(['uploads', 'totalSyncingBytes'], totalSyncingBytes)
-        .setIn(['uploads', 'endEstimate'], endEstimate || undefined)
+      return (
+        state
+          // $FlowFixMe
+          .setIn(['uploads', 'syncingPaths'], I.Set(syncingPaths))
```

Comment:
Is this needed with new immutable?

Reply:
You mean the $FlowFixMe part? It came up when I added more `setIn`s. And since it didn't complain as it does when $FlowFixMe is unnecessary, after a rebase, I assume it means this is still needed. 

Or do you mean something else?

Reply:
I'm gonna merge this for now, and will address more feedbacks in the next PR.